README.md

```diff
@@ -124,3 +124,9 @@ Once in the web app:
 If you see this error while running `azd deploy`: `read /tmp/azd1992237260/backend_env/lib64: is a directory`, then delete the `./app/backend/backend_env folder` and re-run the `azd deploy` command.  This issue is being tracked here: https://github.com/Azure/azure-dev/issues/1237
 
 If the web app fails to deploy and you receive a '404 Not Found' message in your browser, run 'azd deploy'. 
+
+The most common reasons this sample fails are:
```

Comment:
Combine this list with the items above.

Reply:
the items above seem to be random errors, not the top errors we see causing customers to fail here. Seems like two different lists to me, top issues. other issues.